server/controllers/wazuh-utils/wazuh-utils.ts

```diff
@@ -108,7 +108,7 @@ export class WazuhUtilsCtrl {
   async getAppLogs(context: RequestHandlerContext, request: KibanaRequest, response: KibanaResponseFactory) {
     try {
       const lastLogs = await read(
-        WAZUH_DATA_LOGS_RAW_PATH,
+        WAZUH_DATA_LOGS_RAW_FILENAME,
         50
```

Comment:
What does this value refer to? I would add it as a constant where its name helps us know what it is for.